### PR TITLE
HW3_AndriyPovzun

### PR DESCRIPTION
**Pull request by AndriyPovzun**

### CODE REVIEW

File: Andriy_Povzun/Task1.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+employees_eleks = ['John Jones', 'Andy Ruis', 'Oleksandr Usyk',
+                   'Andrii Yevtushenko', 'Will Smith', 'Conor McGregor']
+employees_toshiba = ['Dana White', 'Kamaru Usman', 'George St-Pierre',
+                     'John Jones', 'Antony Joshua', 'Muhammad Ali']
+employees_toshiba.extend(employees_eleks)
+print(employees_toshiba)
```
Comment (bohdandovbsyh) [inline Andriy_Povzun/Task1.py:7]: No new line at the end.
Eleks still have employees , please fix it

File: Andriy_Povzun/Task2.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+                    'Dru Dober', 'Conor McGregor', 'Johny Depp']
+
+poker_blacklist = ['Kamaru Usman', 'John Dodson', 'Tyson Fury',
+                   'Johny Depp', 'Conor McGregor', 'Bradly Cooper']
+
+alcohol_blacklist = ['Mike Tyson', 'Conor McGregor', 'Tyson Fury',
+                     'Johny Depp', 'Tony Ferguson', 'Dana White']
+
+all_blacklist = list(set(casino_blacklist) & set(poker_blacklist) & set(alcohol_blacklist))
+print(f'This humans in all blacklists: {all_blacklist}')
```
Comment (bohdandovbsyh) [inline Andriy_Povzun/Task2.py:11]: add new line to the end

File: Andriy_Povzun/Task3.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+omnivores_guests = ['John', 'Lui', 'Andy', 'Sandy', 'Cler', 'Tanya',
+                    'Fitch', 'Artur', 'Susan', 'Killian']
+vegetarians_guests = ['John', 'Lui', 'Andy', 'Sandy', 'Cler', 'Tanya',
+                      'Fitch', 'Artur', 'Susan', 'Killian', 'Yulia',
+                      'Steve', 'Gleb', 'Alex']
+only_vegetarians = list(set(vegetarians_guests) - set(omnivores_guests))
+print(f'This people eat only vegetarians food: {only_vegetarians}')
+
```
Comment (bohdandovbsyh) [inline Andriy_Povzun/Task3.py:6]: but omnivores can eat vegetables as well

File: Andriy_Povzun/Task4.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+vip_box = ('Andy Ruis', 'Mike Tyson', 'George St-Pierre',
+           'Oleksandr Usyk', 'Tomas Brady', 'Will Smith')
+common_room = ['Conor McGregor', 'Jose Aldo', 'Lionel Messi',
+               'Bradly Cooper', ' Tom Hardy', 'Johny Depp']
+print(f'Vip guests: {vip_box}. \n'
+      f'Guests in common room: {common_room}.')
```
Comment (bohdandovbsyh) [inline Andriy_Povzun/Task4.py:6]: Good. Add new line to the end please

File: Andriy_Povzun/Task5.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+non_unique_names = ['Conor Mcgregor', 'Kamaru Usman', 'John Jones', 'Daniel Cormier',
+                    'Jose Aldo', 'Conor Mcgregor', 'Jose Aldo', 'Jimmy Manuva', 'John Jones']
+unique_names = []
+for name in non_unique_names:
+    if name not in unique_names:
+        unique_names.append(name)
+print(unique_names)
```
Comment (bohdandovbsyh) [inline Andriy_Povzun/Task5.py:5]: please use methods from the lesson , not loops

**Review by bohdandovbsyh — APPROVED** [review]

**Review by bohdandovbsyh — APPROVED** [review]

File: Andriy_Povzun/Task1.py (hunk 1 above)
Comment (AndriyPovzun) [reply Andriy_Povzun/Task1.py:7]: Done.

File: Andriy_Povzun/Task2.py (hunk 2 above)
Comment (AndriyPovzun) [reply Andriy_Povzun/Task2.py:11]: Done.

File: Andriy_Povzun/Task3.py (hunk 3 above)
Comment (AndriyPovzun) [reply Andriy_Povzun/Task3.py:6]: Done.

File: Andriy_Povzun/Task4.py (hunk 4 above)
Comment (AndriyPovzun) [reply Andriy_Povzun/Task4.py:6]: Done.

File: Andriy_Povzun/Task5.py (hunk 5 above)
Comment (AndriyPovzun) [reply Andriy_Povzun/Task5.py:5]: Done.

**Pull request merged**